helm/alfresco-content-services/23.N_values.yaml

```diff
@@ -75,3 +75,8 @@ alfresco-connector-ms365:
 alfresco-connector-msteams:
   image:
     tag: 2.0.0
+dtas:
+  config:
+    assertions:
+      acs:
+        version: 23.1.1
```

Comment:
Shouldn't we rely or at least default to `.Chart.appVersion`?

Reply:
for this stuff I was thinking to just add support for updatecli later

Reply:
well relying on appVersion would avoid having having to create and maintain more updatecli pipelines.

Reply:
I can add it as default in the template but I would keep the value as eventually it would need to be overridden for -A.x/-M.x versions

Reply:
ok no worries, then let's move on as is